client/packages/executor/src/utils/types.ts

```diff
@@ -0,0 +1,161 @@
+import { TextDecoder } from "util";
```

Comment:
What is the purpose of this file? Arent these things part of the SideEffect class?